docker-compose.dev.yml

```diff
@@ -36,12 +37,11 @@ services:
       - "${POSTGRES_PORT:-5432}:5432"
 
   file-server:
+    build:
+      args:
+        NODE_ENV: development
     ports:
       - ${STATIC_FILES_PORT:-8080}:${STATIC_FILES_PORT:-8080}
-    volumes:
-      # This static volume will become more useful when the client apps are configured to run
-      # in development mode.
-      - build:/app/build
```

Comment:
I also missed this; I think this may have been something we were considering for getting the build artifacts out of the containers. Can revisit later.

Reply:
Volumes tend to be sticky in ways that containers are not.
Since this line was creating an unnamed persistent volume with build artifacts in it, it's just the kind of thing that might be culpable in some of the "spooky caching" scenarios we've seen. Eliminating it removes the mystery.
It was serving no practical purpose now, but was stubbed out here as a kind of placeholder for a thing we haven't wired up yet: live serving multiple client apps (via Docker) with hot reloading. I assumed that the first step towards that goal would be this shared build volume. But this needs more consideration.